recipes-support/swupdate/swupdate/raspberrypi3/swupdate

```diff
@@ -0,0 +1,191 @@
+#! /bin/sh
```

Comment:
Will this play nicely with the systemd setup we have? I don't remember if systemd generates a wrapper for such scripts or not.